lib/Graph/Nodes.cpp

```diff
@@ -1172,6 +1172,22 @@ bool BatchOneHotNode::verify() const {
   return isValid;
 }
 
+bool SpaceToDepthNode::verify() const {
+  auto inputN = getInput();
+  auto resultN = getResult();
+  auto inputDims = inputN.dims();
+  auto outputDims = resultN.dims();
+  unsigned blockSize = getBlockSize();
+
+  bool sameType = checkTypeIgnoreShape(inputN, resultN, this);
+  bool dimTransform = inputDims[0] == outputDims[0] &&
+                      inputDims[1] / blockSize == outputDims[1] &&
```

Comment:
```suggestion
                      inputDims[1] == outputDims[1] * blockSize &&
```
(that way we don't have to check if the remainder is 0)